### ISSUE TITLE
Dedicated page(s) for more detailed stats (per repos instance and per host) ?

### ISSUE THREAD

**Issue by bzg** [ecosyste-ms/repos#680]
The overview on repos.ecosyste.ms is this:

![Screenshot 2024-10-09 at 14-14-52 Ecosyste ms Repos](https://github.com/user-attachments/assets/00fc3006-e037-41a0-a0bd-b04a519333cb)

It would be nice to add a link to a page with detailed stats for the whole repos instances and a link for detailed stats for each hosts, and to expose stats on the API.

- X most active repos the last 10 days
- X repos with the most contributors
- X repos with the most subscribers
- X repos with the most stargazers
- X repos with the more forks
- X most active groups the last 10 days
- X groups with the most contributors
- X groups with the most subscribers
- X groups with the most stargazers
- X groups with the more cumulated forks

See https://docs.forge.apps.education.fr/console/ that has been done for the purpose of one GitLab instance.

![2024-10-09-142213_1920x1200_scrot](https://github.com/user-attachments/assets/09df6d25-4f1a-4a0c-ac26-9ab23c00a5b1)

cc @simkim FYI

**Comment by andrew** [comment]
Yeah I think this is a good idea, there are some stats in the api at a host level, the home page summary is just a sum of them across all hosts.

I just need to be a bit performance sensitive with some of the calculations on the very large hosts as the queries could take a long time, but they can always be disabled on repos.ecosyste.ms and enabled for you.
